src/compiler/scala/tools/nsc/settings/ScalaSettings.scala

```diff
@@ -27,7 +27,7 @@ trait ScalaSettings extends AbsScalaSettings
    *  defaults to the value of CLASSPATH env var if it is set, as in Java,
    *  or else to `"."` for the current user directory.
    */
-  protected def defaultClasspath = sys.env.getOrElse("CLASSPATH", ".")
+  protected def defaultClasspath = Option(System.getenv("CLASSPATH")).getOrElse(".")
```

Comment:
I understand the goal of this PR is to cut dependencies, so this comment is out of scope: I'm in favor of shipping this kind of convenience (`envOrElse`) with Scala (in a form that makes it easily available, I understand we want to provide an opt-out).